### PR TITLE
feat(capture-sdk):

### PR DESCRIPTION
**Pull request by AdnanYupi**
- Camera migration guide

PIA-2968

### CODE REVIEW

File: capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst [hunk 1]
```diff
@@ -224,6 +224,19 @@ For example if you need to show animated illustrations you can use a `Lottie
...
+
+New Features
+++++++++++++
+
+We implemented image cropping. Parts of the image that appears outside the white camera frame will be cut out from the final image.
+
+Breaking Changes
+++++++++++++++++
+
+Help tips are removed.
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst:238]: [Suggestion] 
```suggestion
We removed the tooltip popups that were shown on first launch.
```

File: capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst [hunk 2]
```diff
@@ -224,6 +224,19 @@ For example if you need to show animated illustrations you can use a `Lottie
 
 You can find more details `here <customization-guide.html>`_.
 
+Camera
+~~~~~
+
+New Features
+++++++++++++
+
+We implemented image cropping. Parts of the image that appears outside the white camera frame will be cut out from the final image.
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst:233]: [Request] Here we should also mention the bottom navigation bar and activity indicator customisation. In the same way we did it for Help and Analysis.

**Review by a-szotyori — CHANGES_REQUESTED** [review]

File: capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst (hunk 1 above)
Comment (AdnanYupi) [reply capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst]: hahaha

File: capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst (hunk 2 above)
Comment (a-szotyori) [reply capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst:233]: Hey, @AdnanYupi, please also add info bout the activity indicator which is shown during image processing (cropping).

**Review by a-szotyori — APPROVED** [review]
Thank you!

**Pull request merged**